src/components/sidebarMenu/SidebarMenu.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import logo from '../../assets/img/logo.svg';
+import './SidebarMenu.scss'
+import { SidebarMenuItem } from '../sidebarMenuItem/SidebarMenuItem';
+import { SidebarMenuItemFooter } from '../sidebarMenuItemFooter/SidebarMenuItemFooter';
+import ItemList from '../../interfaces/itemMenu.interface';
+
+export const SidebarMenu = () => {
+  const sidebarMenu:ItemList[]=[
+    {
+      id: '1',
+      titleMenu: 'Home',
+    },
+    {
+      id: '2',
+      titleMenu: 'My Files',
+    },
+    {
+      id: '3',
+      titleMenu: 'Recent Files',
+    },
+    {
+      id: '4',
+      titleMenu: 'Shared Files',
+    },
+    {
+      id: '5',
+      titleMenu: 'File Request',
+    },
+    {
+      id: '6',
+      titleMenu: 'Trash',
+    },
+  ]
+  const sidebarMenuFooter:ItemList[]=[
+    {
+      id: '1',
+      titleMenu: 'Upload files',
+    },
+    {
+      id: '2',
+      titleMenu: 'Upload folder',
+    },
+    {
+      id: '3',
+      titleMenu: 'New folder',
+    },
+    {
+      id: '4',
+      titleMenu: 'More',
+    },
+  ]
+  return (
+    <div className="sidebar">
+      <div className="sidebar__logo">
+        <img
+          src={logo}
+          alt="ravnIcon"
+        />
+      </div>
+      <div className="sidebar__menu">
+        {sidebarMenu.map(({titleMenu})=>(
+          <SidebarMenuItem titleMenu={titleMenu} />
+        ))}
+      </div>
+      <div className="sidebar__footer">
+        <div className="sidebar__footer--list">
```

Comment:
I'm curious, why did you decide to name your classes with dashes (`--`) and underscrores? (`__`)?

Reply:
I have been trying to use the (Block Element Modifier) BEM nomenclature for styles